libraries/model-networking/src/model-networking/ModelCache.cpp

```
@@ -72,11 +72,15 @@ void GeometryReader::run() {
                 const bool grabLightmaps = true;
                 const float lightmapLevel = 1.0f;
                 fbxgeo = readFBX(_data, _mapping, _url.path(), grabLightmaps, lightmapLevel);
+                if (fbxgeo->meshes.size() == 0 && fbxgeo->joints.size() == 0) {
+                    // empty fbx geometry, indicates error
+                    throw QString("empty geometry, possibly due to an unsupported FBX version");
```

wait... do we really want to throw an exception here? Isn't that essentially the same as a crash later?

Wouldn't it be better to report an error, and then treat the resource as if it was an empty mesh?